src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -991,9 +996,14 @@ private bool ShouldCheckConstraints
 
         internal NamedTypeSymbol GetSpecialType(SpecialType typeId, DiagnosticBag diagnostics, CSharpSyntaxNode node)
         {
+            return GetSpecialType(typeId, diagnostics, node.Location);
```

Comment:
Usually, there is always a syntax node out there, in case of a lambda parameter too.